backend/webapi/Features/AccessRequests/AccessRequestsController.cs

```diff
@@ -83,6 +83,15 @@ public async Task<IActionResult> CreateImmsBCEformsEnrolment([FromServices] ICom
         => await this.AuthorizePartyBeforeHandleAsync(command.PartyId, handler, command)
             .ToActionResult();
 
+    [HttpPost("immsbc")]
+    [Authorize(Policy = Policies.HighAssuranceIdentityProvider)]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    public async Task<IActionResult> CreateImmsBCEnrolment([FromServices] ICommandHandler<ImmsBC.Command, IDomainResult> handler,
+                                                                 [FromRoute] ImmsBC.Command command)
```

Comment:
Very small formatting request: could you please align the square brackets on this line like the other endpoints